### PR TITLE
Merge Flagger Roadmap items

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #904

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: content/en/roadmap.md [hunk 1]
```diff
@@ -183,3 +187,16 @@ Tasks
 - [x]  <span style="color:grey">Implement support for referring to an alternative chart values file</span>
 - [x]  <span style="color:grey">Stabilize API</span>
 - [x]  <span style="color:grey">[Create a migration guide for Helm Operator users](docs/migration/helm-operator-migration.md)</span>
+
+## Flagger
+
+### [GitOps Toolkit](https://github.com/fluxcd/flux2) compatibility
+
+- [ ] Migrate Flagger to Kubernetes controller-runtime and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+- [ ] Make the Canary status compatible with [kstatus](https://github.com/kubernetes-sigs/cli-utils)
```
Comment (aryan9600) [inline content/en/roadmap.md:196]: These two points don't fall under GOTK compatibility, I think the "Integrations" section might be more apt.

Comment (dholbach) [reply content/en/roadmap.md:196]: I simply copied from https://github.com/fluxcd/flagger/blob/main/README.md#roadmap :grimacing:

Comment (aryan9600) [reply content/en/roadmap.md:196]: oh I see, well that's wrong then I guess. I don't see how `kubebuilder` usage and `kstatus` compatibility fall under GOTK.
Maybe we should have another section named "Kubernetes Ecosystem" for these two?

**Review by stefanprodan — CHANGES_REQUESTED** [review]
Please hold this. We need to rethink the whole Flagger & Flux integration.

File: content/en/roadmap.md (hunk 1 above)
Comment (stefanprodan) [reply content/en/roadmap.md:196]: @aryan9600 kstatus compatibility is needed for Flux to be able to wait for Canaries.

**Pull request closed without merging**